src/translate/pddl/conditions.py

```diff
@@ -220,7 +226,7 @@ class Literal(Condition):
     __hash__ = Condition.__hash__
     parts = []
     __slots__ = ["predicate", "args", "hash"]
-    def __init__(self, predicate, args):
+    def __init__(self, predicate: str, args: List[str]) -> None:
```

Comment:
Is it common to add the return type for __init__ functions? Aren't those almost always None?

Reply:
I followed the examples in the documentation at python.org.  They make the None explicit.

Reply:
Interesting, thanks!